src/dsp_tools/utils/xmlupload/xmlupload.py

```diff
@@ -482,12 +404,14 @@ def _handle_upload_error(
     if stash:
         filename = _save_stash_as_json(
             stash=stash,
-            save_location=save_location,
-            timestamp_str=timestamp_str,
+            save_location=diagnostics.save_location,
+            timestamp_str=diagnostics.timestamp_str,
         )
+        logfile = Path.home() / ".dsp_tools" / "logging.log"
```

Comment:
The cleaner way would be to retrieve this info with logger.handlers[0].baseFilename.

Reply:
But why do you print this info inside `if stash`? I would print it always